### PR TITLE
DOCS-14953: scala compatability tables

### PR DESCRIPTION
**Pull request by ccho-mongodb**
## Pull Request Info

Question for reviewers:
In the existing Language Compatibility table https://docs.mongodb.com/drivers/scala/#language-compatibility, there are two entries for 1.1. Which one is correct? In this PR, I removed the duplicated line and marked driver v 1.1 as compatible with Scala 2.11 and 2.12.

### Issue JIRA link:
https://jira.mongodb.org/browse/DOCS-14953

### Snooty build log:
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=61ba3f3cc52f2a460fe06dae

### Docs staging link (requires sign-in on MongoDB Corp SSO):

MongoDB compatibility: https://docs-mongodbcom-staging.corp.mongodb.com/drivers/docsworker-xlarge/DOCS-14953-scala-compatibility-tables/scala/#mongodb-compatibility

Language compatibility: https://docs-mongodbcom-staging.corp.mongodb.com/drivers/docsworker-xlarge/DOCS-14953-scala-compatibility-tables/scala/#language-compatibility

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [x] Are all the links working?
- [x] Are the staging and workerpool job links in the PR description updated?

### CODE REVIEW

**Comment by stIncMale** [comment]
I see nothing referring to Scala version in the current version in the file `driver-scala/build.gradle`. In the file `build.gradle` there is `implementation ('org.scala-lang:scala-library:%scala-version%')`, but I don't see `scala-version` being defined anywhere. @rozza is probably the only one who can tell how to determine which Scala version the driver is compatible with, specifically the version 1.1 of the driver.

**Review by rozza — CHANGES_REQUESTED** [review]
Added some feedback with the requested changes.

@stIncMale fyi the scala versions we compile against are all in the [gradle.properties](https://github.com/mongodb/mongo-java-driver/blob/master/gradle.properties#L19) file

File: source/includes/language-compatibility-table-scala.rst [hunk 1]
```diff
@@ -8,79 +8,67 @@
...
-
+     - 
+     - ✓
+     - ✓
    * - 2.0
-     -
-     - |checkmark|
-     - |checkmark|
-
-   * - 1.1
```
[the remark this answers is not in the document]
Comment (ccho-mongodb) [reply source/includes/language-compatibility-table-scala.rst:73]: Thanks, I'll add the row back with the correct version.

**Review by rozza — APPROVED** [review]
LGTM!

**Pull request merged**